tests/modules/saml/lib/IdP/SAML2Test.php

```diff
@@ -160,7 +160,7 @@ public function testIdPInitShibCompatOptionalParams()
      */
     private function idpInitiatedHelper(array $queryParams)
     {
```

Comment:
This one slipped through, but it's obviously one UndefinedMethod, one InvalidArgument and the MissingParamType/MissingReturnType